12-breach_policy.md

```diff
@@ -4,9 +4,9 @@ layout: default
 
 # 12. Breach Policy
 
-To provide guidance for breach notification when impressive or unauthorized access, acquisition, use and/or disclosure of the ePHI occurs. Breach notification will be carried out in compliance with the American Recovery and Reinvestment Act (ARRA)/Health Information Technology for Economic and Clinical Health Act (HITECH) as well as any other federal or state notification law.
+To provide guidance for breach notification when impressive or unauthorized access, acquisition, use and/or disclosure of the ePHI occurs. Breach notification will be carried out in compliance with the [American Recovery and Reinvestment Act](https://www.fcc.gov/general/american-recovery-and-reinvestment-act-2009) (ARRA)/Health Information Technology for Economic and Clinical Health Act (HITECH) as well as any other federal or state notification law.
 
-The Federal Trade Commission (FTC) has published breach notification rules for vendors of personal health records as required by ARRA/HITECH. The FTC rule applies to entities not covered by HIPAA, primarily vendors of personal health records. The rule is effective September 24, 2009 with full compliance required by February 22, 2010.
+The Federal Trade Commission (FTC) has published [breach notification rules](https://www.ftc.gov/legal-library/browse/rules/health-breach-notification-rule) for vendors of personal health records as required by ARRA/HITECH. The FTC rule applies to entities not covered by HIPAA, primarily vendors of personal health records. The rule is effective September 24, 2009 with full compliance required by February 22, 2010.
```

Comment:
Is there any reason why we have all of this language? I've seen other policies and they don't go into details about why they are using it and which rules/framework/policies? Trying to under the context.

Reply:
I think this was likely significant 10 years, but generally breach notifiation does need to happen for HIPAA entities.  The policies were sourced from an open-source policy framework that is no longer maintained, as I checked to see if they ever made any updates to it.